src/gitignore.rs

```diff
@@ -137,19 +137,17 @@ impl GitignoreFile {
                 pat = pat + "/**";
             }
 
-            let glob = try!(GlobBuilder::new(&pat)
-                .literal_separator(true)
-                .build());
+            let glob = try!(GlobBuilder::new(&pat).literal_separator(true).build());
 
             builder.add(glob);
             patterns.push(p);
         }
 
         Ok(GitignoreFile {
-            set: try!(builder.build()),
-            patterns: patterns,
-            root: root.to_owned(),
-        })
+               set: try!(builder.build()),
```

Comment:
I think so - at least that's what it has done for me after upgrading to rustfmt-0.8.0...